examples/compat/ggplot/density.py

```diff
@@ -9,4 +9,4 @@
 
 plt.title("Density ggplot-based plot in Bokeh.")
 
-mpl.to_bokeh(name="density")
+mpl.show(mpl.to_bokeh(name="density"))
```

Comment:
> the Python's "magic" import behaviour disappear...

I know that, but that's not the point. The point is that imports are transitive in Python. I shouldn't be able to import an imported name, at least without explicitly marking it as the API of the module where it is imported.